openid-connect-server/src/test/java/org/mitre/openid/connect/service/impl/TestMITREidDataService_1_1.java

```diff
@@ -1207,16 +1207,16 @@ public void testExportGrants() throws IOException {
                 } else {
                     assertThat(site.get("whitelistedSiteId").getAsLong(), equalTo(compare.getWhitelistedSite().getId()));
                 }
-                if (site.get("approvedAccessTokens").isJsonNull() || site.getAsJsonArray("approvedAccessTokens") == null) {
-                    assertTrue(compare.getApprovedAccessTokens() == null || compare.getApprovedAccessTokens().isEmpty());
-                } else {
-                    assertNotNull(compare.getApprovedAccessTokens());
-                    Set<String> tokenIds = new HashSet<String>();
-                    for(OAuth2AccessTokenEntity entity : compare.getApprovedAccessTokens()) {
-                        tokenIds.add(entity.getId().toString());
-                    }
-                    assertThat(jsonArrayToStringSet(site.getAsJsonArray("approvedAccessTokens")), equalTo(tokenIds));
-                }
+//                if (site.get("approvedAccessTokens").isJsonNull() || site.getAsJsonArray("approvedAccessTokens") == null) {
```

Comment:
It's a red flag to me that these unit tests were all commented out and not replaced with equivalents.